src/modules/fancyzones/FancyZonesLib/FancyZones.cpp

```diff
@@ -308,6 +308,10 @@ void FancyZones::MoveWindowIntoZone(HWND window, winrt::com_ptr<IWorkArea> zoneW
     auto& fancyZonesData = FancyZonesDataInstance();
     if (!fancyZonesData.IsAnotherWindowOfApplicationInstanceZoned(window, zoneWindow->UniqueId()))
     {
+        if (zoneWindow)
+        {
+            Trace::FancyZones::SnapNewWindowIntoZone(zoneWindow->ActiveZoneSet());
```

Comment:
Should we log that the window was actually snapped to a zone or just supposed to be snapped?

Reply:
The idea of the event is to get metrics on "Activation", which means: "Are users using PowerToys features actively?"
So it would be valid whether the user snapped to a zone or tried to snap to a zone.

Reply:
I agree. Validating the success/failure of the snapping would also be useful, but we can work on that in a different PR once we're more aligned on overall telemetry metrics.